src/overtone/studio/inst.clj

```diff
@@ -81,16 +110,23 @@
   "Control the volume of a single instrument."
   [inst vol]
   (ensure-node-active! inst)
-  (ctl (:mixer inst) :volume vol)
+  (ctl @(:mixer inst) :volume vol)
   (reset! (:volume inst) vol))
 
 (defn inst-pan!
   "Control the pan setting of a single instrument."
   [inst pan]
   (ensure-node-active! inst)
-  (ctl (:mixer inst) :pan pan)
+  (ctl @(:mixer inst) :pan pan)
   (reset! (:pan inst) pan))
 
+(defn inst-mixer-ctl!
+  "Control a named parameters of the output mixer of a single instrument."
+  [inst & args]
+  (ensure-node-active! inst)
+  (apply ctl @(:mixer inst) args)
```

Comment:
Not for this PR, just to start a discussion, but maybe `ctl` could call an interface that we could implement, for example, for `Inst` so we can get rid of these one-off `inst-...` functions.

Reply:
Very nice idea. `ctl` does the right thing already. `Inst` would need to implement `IControllableNode` and delegate to the mixer node.